cmd/buildkitd/config/gcpolicy_windows.go

```diff
@@ -3,10 +3,36 @@
 
 package config
 
+import "golang.org/x/sys/windows"
+
 func DetectDefaultGCCap() DiskSpace {
-	return DiskSpace{Bytes: defaultCap}
+	return DiskSpace{Percentage: 10}
```

Comment:
As windows images are usually much bigger than Linux ones, feel free to choose a bigger default (and bigger fallback) if you need to.

Reply:
BTW, what do you think about making this configurable for both platforms?

Reply:
It is already configurable with `--...-gc-keepstorage`

Reply:
got it.